feature/service-registration/data/src/main/java/com/pob/umpa/feature/service_registration/data/sheet_music/mapper/SheetMusicMapper.kt

```diff
@@ -0,0 +1,102 @@
+package com.pob.umpa.feature.service_registration.data.sheet_music.mapper
+
+import com.pob.umpa.feature.service_registration.data.sheet_music.model.DeliveryTime
+import com.pob.umpa.feature.service_registration.data.sheet_music.model.DeliveryTimeAverage
+import com.pob.umpa.feature.service_registration.data.sheet_music.model.RevisionInfo
+import com.pob.umpa.feature.service_registration.data.sheet_music.model.SampleSheetModel
+import com.pob.umpa.feature.service_registration.data.sheet_music.model.ServiceDetail
+import com.pob.umpa.feature.service_registration.data.sheet_music.model.ServiceProfile
+import com.pob.umpa.feature.service_registration.data.sheet_music.model.SheetMusicServiceInfoModel
+import com.pob.umpa.feature.service_registration.data.sheet_music.model.SheetMusicServiceProfileModel
+import com.pob.umpa.feature.service_registration.data.sheet_music.model.TimeType
+import com.pob.umpa.feature.service_registration.domain.sheet_music.entity.SampleSheetEntity
+import com.pob.umpa.feature.service_registration.domain.sheet_music.entity.SheetMusicServiceInfoEntity
+import com.pob.umpa.feature.service_registration.domain.sheet_music.entity.SheetMusicServiceProfileEntity
+import com.pob.umpa.feature.service_registration.data.sheet_music.model.Image
+import com.pob.umpa.feature.service_registration.data.sheet_music.model.PricePerPage
+import com.pob.umpa.feature.service_registration.data.sheet_music.model.Sheet
+import com.pob.umpa.feature.service_registration.data.sheet_music.model.SheetType
+
+//fun SheetMusicModel.toSheetMusicServiceInfo() : SheetMusicServiceInfo {
+//
+//}
+//
+//fun SheetMusicModel.toSheetMusicServiceProfile() : SheetMusicServiceProfile {
+//
+//}
+//
+//fun SheetMusicModel.toSampleSheet() : SampleSheet {
+//
+//}
+
+fun SheetMusicServiceInfoEntity.toSheetMusicServiceInfoModel() : SheetMusicServiceInfoModel {
```

Comment:
엔티티와 모델간은 서로 동일한 클래스끼리 변환하는 작업이니 간단하게 .toModel로 메소드 이름을 간단하게 정해도 좋을거 같아용

```suggestion
fun SheetMusicServiceInfoEntity.toModel() : SheetMusicServiceInfoModel {
```

Reply:
toType() 확장함수는 기본적으로 타입을 변환하는 함수이니까, 저는 타입을 명시해주는 편이 낫다고 생각합니다...!

Reply:
> 엔티티와 모델간은 서로 동일한 클래스끼리 변환하는 작업이니 간단하게 .toModel로 메소드 이름을 간단하게 정해도 좋을거 같아용

toType() 확장함수는 기본적으로 타입을 변환해 반환하는 함수를 나타낼 때 사용하는 스타일이니까, 저는 타입을 명시해주는 편이 낫다고 생각합니다...! 다만, 저희 전부 모든 매퍼에 대한 컨벤션을 통합한다면 toModel도 명시적으로 쓸 수 있으니까 회의 후에 다들 괜찮으면 바꿔도 좋을 것 같슴당